types/const.go

```diff
@@ -1,6 +1,5 @@
 package types
 
 const (
-	ArtifactJSONSchemaVersion = 1
-	BlobJSONSchemaVersion     = 1
+	SchemaVersion = 1
```

Comment:
Do you think we should unify these versions? Also, I think this is no longer `SchemaVersion` because we have to update it when we want to invalidate the cache.

Reply:
I think its nice and easy to keep things simple with one version. What's the counter argument?